src/olympia/reviewers/models.py

```diff
@@ -802,7 +802,7 @@ class NeedsHumanReview(ModelBase):
             ),
             (
                 REASON_ABUSE_ADDON_VIOLATION_APPEAL,
-                'Appeal about a decision on abuse reported within the add-on',
+                'Appeal against a reviewer decision about a policy violation',
```

Comment:
nit: "Appeal _of_ a reviewer's decision about a policy violation" sounds a bit more neutral.

Reply:
Ah, thanks, will update. I had looked it up on dict.leo.org and one example there mentioned _against_.